lib/Doctrine/Persistence/Event/LifecycleEventArgs.php

```diff
@@ -61,3 +62,4 @@ public function getObjectManager()
 }
 
 class_exists(\Doctrine\Common\Persistence\Event\LifecycleEventArgs::class);
+interface_exists(\Doctrine\Common\Persistence\ObjectManager::class);
```

Comment:
That one may be useless because it refers to the constructor, no signature compatibility issue should happen here.